boards/st/stm32l562e_dk/stm32l562e_dk_common.dtsi

```diff
@@ -42,6 +44,74 @@
 	};
 };
 
+&fmc {
+	pinctrl-0 = <&fmc_a0_pf0 &fmc_nce_pd7 &fmc_nwe_pd5 &fmc_noe_pd4
+		     &fmc_d0_pd14 &fmc_d1_pd15 &fmc_d2_pd0 &fmc_d3_pd1
+		     &fmc_d4_pe7 &fmc_d5_pe8 &fmc_d6_pe9 &fmc_d7_pe10
+		     &fmc_d8_pe11 &fmc_d9_pe12 &fmc_d10_pe13 &fmc_d11_pe14
+		     &fmc_d12_pe15 &fmc_d13_pd8 &fmc_d14_pd9 &fmc_d15_pd10>;
+	pinctrl-names = "default";
+	status = "okay";
+
+	sram {
+		compatible = "st,stm32-fmc-nor-psram";
+
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		bank@0 {
+			reg = <0x0>;
+			st,control = <STM32_FMC_DATA_ADDRESS_MUX_DISABLE
+				STM32_FMC_MEMORY_TYPE_SRAM
+				STM32_FMC_NORSRAM_MEM_BUS_WIDTH_16
+				STM32_FMC_BURST_ACCESS_MODE_DISABLE
+				STM32_FMC_WAIT_SIGNAL_POLARITY_LOW
+				STM32_FMC_WAIT_TIMING_BEFORE_WS
+				STM32_FMC_WRITE_OPERATION_ENABLE
+				STM32_FMC_WAIT_SIGNAL_DISABLE
+				STM32_FMC_EXTENDED_MODE_DISABLE
+				STM32_FMC_ASYNCHRONOUS_WAIT_DISABLE
+				STM32_FMC_WRITE_BURST_DISABLE
+				STM32_FMC_CONTINUOUS_CLOCK_SYNC_ONLY
+				STM32_FMC_WRITE_FIFO_DISABLE
+				STM32_FMC_PAGE_SIZE_NONE>;
+			st,timing = <1 1 32 0 2 2 STM32_FMC_ACCESS_MODE_A>;
+
+			fmc-mipi-dbi {
+				compatible = "st,stm32-fmc-mipi-dbi";
+				reset-gpios = <&gpiof 14 GPIO_ACTIVE_LOW>;
+				power-gpios = <&gpioh 0 GPIO_ACTIVE_LOW>;
+				#address-cells = <1>;
+				#size-cells = <0>;
+				st7789v: lcd-panel@0 {
+					compatible = "sitronix,st7789v";
+					reg = <0>;
+					mipi-mode = <MIPI_DBI_MODE_8080_BUS_16_BIT>;
+					mipi-max-frequency = <0>;
```

Comment:
See my comment within the driver- this parameter should be calculated based off the datarate of the bus